README.md

```diff
@@ -1,123 +1,73 @@
-
-```sh
-$ gwa login --help
-You can login via device login or by using client credentials
-
-To use device login, simply run the command like so:
-    $ gwa login
-
-To use your credentials you must supply both a client-id and client-secret:
-    $ gwa login --client-id <YOUR_CLIENT_ID> --client-secret <YOUR_CLIENT_SECRET>
-...
-
-```
-
-To generate a Markdown-formatted string of every command's documentation to `stdout`, run `$ just docs`. You could then pipe it to a file like so `$ just docs > gwa-commands.md`, though this will be just a temporary solution.
-
-## Development
-
-Prerequisites:
-- [Go](https://go.dev) 1.20 or higher
-- [Just](https://github.com/casey/just) (alternative to `make`)
-- [GoReleaser](https://goreleaser.com)
-
-Tools:
-- [Cobra] Command line argument parser
-- [Viper](https://github.com/spf13/viper) Configuration file manager, integrates tightly with Cobra
-- [Lipgloss](https://github.com/charmbracelet/lipgloss) Styles and colours
-
-#### Steps to set up dev environment
-
-1. Verify you have Go 1.20+ installed
-
-   ```sh
-   $ go version
-   ```
-   If you don't have `go` installed on your machine, follow instructions on [the Go website](https://golang.org/doc/install).
-
-2. Clone this repository
-
-   ```sh
-   $ git clone git@github.com:bcgov/gwa-cli.git
-   $ cd gwa-cli
-   $ mv env.example .env
-   ```
-   **Note** Some local environments require Go projects are run from the `$HOME/go/src` directory. If any `module not found` errors are reported, try moving it.
-
-   Also be sure to fill in the environment variables before running a command.
-
-3. Run commands
+## Installation
+To get started with `gwa`, follow the detailed instructions provided in the DevHub [how-to](https://dev.developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/how-to/gwa-install/) guide. This guide covers all the steps required to install and configure the tool on your system.
 
-   Test any commmands by running `just run` in the `cwd`. You can also use `$ just test` to run all tests.
+---
 
-   ```sh
-   $ just run gateway current
-   your-gateway
-   $ just test
-   ?   	github.com/bcgov/gwa-cli	[no test files]
-   ok  	github.com/bcgov/gwa-cli/cmd
-   ok  	github.com/bcgov/gwa-cli/pkg
-   ```
+## Usage
 
-4. Set up your IDE
+### Use `--help`
 
-   Go has great tooling which is required to ensure code contributed is formatted consitently and is type-safe.
+To make the most out of the `gwa` tool, you can use the `--help` command to get detailed information on its various commands and options. Here's how you can use it:
 
-   - **VSCode:** Install the [official Go extension](https://marketplace.visualstudio.com/items?itemName=golang.Go)
-   - **NeoVim:** [go.nvim](https://github.com/ray-x/go.nvim) is a great plugin
+#### Basic Usage of `--help`
+The `--help` flag can be used with any command within the `gwa` tool to display information about that command, including its purpose, options, and examples.
 
```

Comment:
```suggestion
```
This is redundant with the first sentence in this section. Avoid unnecessary deeper-level headers